server/src/main/java/org/apache/iotdb/db/service/thrift/ThriftService.java

```diff
@@ -45,6 +45,11 @@
 
   private CountDownLatch stopLatch;
 
+  // whether enable the service. By default it is true.
+  // If we just want to create an IoTDB instance without RPC service, set this false before call
+  // IoTDB.active()
+  private boolean enable = true;
```

Comment:
It seems that we should initialize this field with `enableRpcService` in `Config`?

Reply:
Thanks, actually, this is not used, I originally thought this was used by the single module, after reviewing the codes, it is useless, I will remove this attribute.

Whether enable the single module's RPC port is controlled by the `enableRpcService` in IoTDBConfig.